src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -370,6 +371,9 @@ def list_workflow_runs(
             workflow_def = self.get_workflow_def(r.definitionId)
             workflow_runs.append(r.to_ir(workflow_def))
 
+        if limit is not None:
+            workflow_runs = workflow_runs[-limit:]
```

Comment:
Is there a way we can use CE's API to avoid client-side filtering?

Reply:
FYI: the API spec is here: https://github.com/zapatacomputing/workflow-driver/blob/main/openapi/src/resources/workflow-runs.yaml

Reply:
As of [d7092e1](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/111/commits/d7092e1838efbd4bc9432e549773e8f61a9510f9), requests the precise number of workflow runs as are needed.